Docs/Getting Started Guide.md

```diff
@@ -152,28 +140,29 @@ struct SetOAuthURL: Action {
+Reducers are the **only place** in which you should **modify** application state. Reducers take the current application state and an action then return the new transformed application state. We recommend to provide many small reducers that each handle a subset of your application state.
 
-You can do this implementing a top-level reducer that conforms to the `Reducer` protocol. This reducer will then call individual functions for each different part of the app state.
+You can do this by implementing a top-level reducer that conforms to the `Reducer` protocol. This reducer will then call individual reducer functions for each different part of the app state, according to their areas of responsibility.
 
-Here's an example in which we construct a new state, by calling sub-reducers with different sub-states:
+Here's an example in which we construct a new state by calling sub-reducers with different sub-states:
 
 ```swift
 func appReducer(action: Action, state: State?) -> State {
     return State(
-      navigationState: navigationReducer(action, state: state?.navigationState),
-      authenticationState: authenticationReducer(state?.authenticationState, action: action),
-      repositories: repositoriesReducer(state?.repositories, action: action),
-      bookmarks: bookmarksReducer(state?.bookmarks, action: action)
+      navigationState: navigationReducer(action, state?.navigationState),
+      authenticationState: authenticationReducer(action, state?.authenticationState),
+      repositories: repositoriesReducer(action, state?.repositories),
+      bookmarks: bookmarksReducer(action, state?.bookmarks)
    )
 }
 ```
-The `Reducer` typealias is a method that takes an `Action` and an `State?` and returns a `State`. Typically reducers will be responsible for initializing the application state. When they receive `nil` as the current state, they should return the initial default value for their portion of the state. In the example above the `appReducer` delegates all calls to other reducer functions. E.g. the `authenticationReducer` is responsible for providing the `authenticationState`.
 
-Here's what the `authenticationReducer` function that is called from the `appReducer` looks like:
+The `Reducer` typealias is a function that takes an `Action` and a `State?` then returns a `State`. Typically reducers will be responsible for initializing the application state. When they receive `nil` as the current state, they should return the initial default value for their portion of the state. In the example above the `appReducer` delegates all calls to other reducer functions. 
+
+For example, `authenticationReducer` is responsible for providing the `authenticationState`. Here's what the `authenticationReducer` function might look like:
 
 ```swift
-func authenticationReducer(state: AuthenticationState?, action: Action) -> AuthenticationState {
+func authenticationReducer(_ state: AuthenticationState?, _ action: Action) -> AuthenticationState {
```

Comment:
To match the call above (Line 152), this would have to be `authenticationReducer(_ action: Action, _ state: AuthenticationState)`. I prefer named parameter, to be frank, though.